debian/changelog

```diff
@@ -1,4 +1,4 @@
-cloud-init (23.4.1-0ubuntu1~23.10.3) UNRELEASED; urgency=medium
+cloud-init (24.1.1-0ubuntu1~23.10.1) mantic; urgency=medium
```

Comment:
Looks like our [tooling doesn't yet handle a shift from major.minor.hotfix in d/changelog to a new commit based on just major.minor](https://github.com/canonical/uss-tableflip/pull/119). We need to manually drop this patch version as this is our first release of 24.1 to mantic and our tooling wasn't aware of the shift to the next major.minor so it left the trailing patch in place.
```suggestion
cloud-init (24.1-0ubuntu1~23.10.1) mantic; urgency=medium
```


```suggestion
cloud-init (24.1-0ubuntu1~23.10.1) mantic; urgency=medium
```